src/status_im2/contexts/chat/home/view.cljs

```diff
@@ -55,27 +56,29 @@
 
 (defn tabs []
   (let [selected-tab (reagent/atom :recent)]
-    (fn []
-      (let [contact-requests (rf/sub [:activity.center/notifications-contact-requests])]
-        [:<>
-         [quo/discover-card {:title       (i18n/label :t/invite-friends-to-status)
-                             :description (i18n/label :t/share-invite-link)}]
-         [quo/tabs {:style          {:margin-left   20
-                                     :margin-bottom 20
-                                     :margin-top    24}
-                    :size           32
-                    :on-change      #(reset! selected-tab %)
-                    :default-active @selected-tab
-                    :data           [{:id    :recent
-                                      :label (i18n/label :t/recent)}
-                                     {:id    :groups
-                                      :label (i18n/label :t/groups)}
-                                     {:id                :contacts
-                                      :label             (i18n/label :t/contacts)
-                                      :notification-dot? (pos? (count contact-requests))}]}]
-         (if (= @selected-tab :contacts)
-           [contacts contact-requests]
-           [chats @selected-tab])]))))
+    [:f>
+     (fn []
+       (react/effect! #(rf/dispatch [:activity-center.notifications/fetch-unread-contact-requests]))
```

Comment:
> honestly, I'm not sure what's the point of removing and modifying code in the old codebase.

removing is a good thing because it will prevent devs from just modifying the old code. I agree about the modifying, we have to keep the migration to status_im2 a prioritisation. imo if a follow up p.r is easier for situations like this then that's also fine

Reply:
> honestly, I'm not sure what's the point of removing and modifying code in the old codebase.

I'm removing because there's no point in having dead code in the codebase, we have git for that. It adds burden to maintenance for no added benefit and confuses the heck of all contributors. At least that's my take on the problem @flexsurfer.

Reply:
> could you please elaborate on why it's here as an effect? i guess we have a signal for new ? and we need to only once fetch after sign in then better to move this to event after sign in

I'll give it another try @flexsurfer, that was my first approach, but perhaps something else wasn't working and it confused me. I'll get back to this thread soon to let you guys know.

Reply:
I changed the code to use signals to keep the app db synchronized with pending contact requests. I have no clue why I implemented https://github.com/status-im/status-mobile/pull/14533/commits/e438f3a0bad677e1e5bae24aa3cb79d85f5c3eda in the first place.

Thanks for spotting the issue.